modules/system/console/asset/exceptions/PackageIgnoredException.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
```

Comment:
@LukeTowers what do you think about all these exceptions?

Reply:
@jaxwilko eh, does it serve a purpose having custom exceptions or would it be better to just have custom messages on a SystemException? What's actually using them? Less code = more betterer